src/pal/inc/pal_corefx.h

```diff
@@ -45,6 +45,31 @@ ForkAndExecProcess(
            int* stdoutFd,
            int* stderrFd);
 
+struct corefx_stat {
+    int64_t dev;     /* ID of device containing file */
+    int32_t mode;    /* protection */
+    int32_t uid;     /* user ID of owner */
+    int32_t gid;     /* group ID of owner */
+    int64_t size;    /* total size, in bytes */
+    int64_t atime;   /* time of last access */
+    int64_t mtime;   /* time of last modification */
+    int64_t ctime;   /* time of last status change */
+};
```

Comment:
Yeah, or just have it be 0, though a Boolean value is more reliable.  Right now we effectively just return 0:
https://github.com/dotnet/corefx/blob/master/src/System.IO.FileSystem/src/System/IO/UnixFileSystemObject.cs#L151
but we might want to change that to throw a PlatformNotSupportedException if it's not available.. I'm not sure what behavior we want for that case.